library/Fission/User/Mutation/Error.hs

```diff
@@ -15,6 +18,8 @@ data Create = FailedDigest
 
 instance Display Create where
   display FailedDigest = "Could not create password digest"
+  display AlreadyExists = "The username or email already exists in our system"
 
 instance ToServerError Create where
-  toServerError FailedDigest = err500 { errBody = "Could not create password digest" }
+  toServerError FailedDigest = err500 { errBody = UTF8.showLazyBS <| FailedDigest }
+  toServerError AlreadyExists = err409 { errBody = UTF8.showLazyBS <| AlreadyExists }
```

Comment:
Yeah, just like in Elixir, let's not over-pipe We're just calling a single function on lines 24 & 25

Reply:
yup good call